index.html

```diff
@@ -0,0 +1,148 @@
+            <rect y="30" width="100" height="20"></rect>
+            <rect y="60" width="100" height="20"></rect>
+        </svg>
+        <nav class="navbar desktop">
+            <div id="logo">
+                <h3 class="logo">
+                    Classical <span id='logi'>|Fest</span>
+                </h3>
+                <span class="close-btn" id="close-btn">&times;</span>
+            </div>
+
+            <ul class="navItems desktop" id="navBarr">
+                <li><a id="special" href="#">Home</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main id="man">
+        <section id="home">
+            <article id="Introduction">
+                <h1 class="page-header"><span class="spani">Classical</span><br> Music <span class="spani">Festival</span>
+                    2022
+                </h1>
+                <p>"The one and Only"</p>
+
+            </article>
+            <div id="context">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi rem architecto libero quis repellat
+                    autem </p>
+            </div>
+            <article id="role">
+                <h2>16.11.2022 (THU) ~ 18 (SAT)</h2>
+                <p>@ Jinnah Stadium Karachi , Sindh </p>
+            </article>
+        </section>
+        <section class="program">
+            <div class = "Center-grid">
+                <h2 id="type">Program</h2>
+                <div class="line"></div>
+            </div>
+            <div class="program-container">
+                <article class="program-item para1">
+                    <img src="icons/music.png" alt="Classical Music">
+                    <div>
+                        <h3 class="clrMain">Day 1</h3>
+                        
+                    </div>
+                    <p class="fs-main">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi rem architecto libero quis
+                        </p>
+                </article>
+                <article class="program-item para2">
+                    <img src="icons/Photography.png" alt="Photography">
+                    <div>
+                        <h3 class = "clrMain">Day 2</h3>
+                        
+                    </div>
+                    <p class="fs-main">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi rem architecto libero quis
+                        </p>
+                </article>
+                <article class="program-item para3">
+                    <img src="icons/Fireworks.png" alt="Festival">
+                    <div>
+                        <h3 class = "clrMain">Day 3</h3>
+                        
+                    </div>
+                    <p class="fs-main">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi rem architecto libero quis
```

Comment:
- [ ] Kindly make sure you have 6 programs in this section just like in the original design 

Your design  | the template
----|-----
![image](https://user-images.githubusercontent.com/65068771/181663271-3fd9b836-a758-43ce-8548-a1ced4187c8d.png) | ![image](https://user-images.githubusercontent.com/65068771/181663331-a0002ff9-164d-4fe0-9f7c-d1fe2c15b45b.png)

- [ ] Also make sure you are using the same front-size, padding on texts just like in the original design